src/scripting_api/code_gen.js

```diff
@@ -0,0 +1,107 @@
+/* Copyright 2020 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint no-var: error */
```

Comment:
Rather than adding this in every new file, please add an `.eslintrc` file in this directory instead.
The following file-contents should suffice:
```
{
  "extends": [
    "../../.eslintrc"
  ],

  "rules": {
    // ECMAScript 6
    "no-var": "error",
  },
}
```